### ISSUE TITLE
Can't run python script

### ISSUE THREAD

**Issue by sinewave440hz** [NuclearLighthouseStudios/Cassette-Video-Encoder#1]
Getting this error: 

File "./enc.py", line 131
    print("\rProcessing {}/{} frames".format(count+1,len(images)), end='')

One thing that I didn't do is run the script in a virtual environment. If that's the likely problem, could you give me a pointer on how to go about that? Should I create a vm? Or just install python 3? Total noob to this, as I'm sure you can tell :)

**Comment by NuclearLighthouseStudios** [comment]
As long as you have all the requirements installed the virtual environment shouldn't matter, it just helps keep things cleaner. You can read up about it here: https://virtualenv.pypa.io/en/stable/

The script definitely needs python 3 though. I've just updated the script to make sure it will always prefer python 3 if it's installed. When you're running the script using python 3, make sure to also install the dependencies for python 3 instead of 2, in case python 2 is still the default  on your operating system.

**Comment by sinewave440hz** [comment]
Cool, thanks for the pointers. I'm on Mac, so the default python was too old. I ended up installing pyenv, according to [this guide.](https://opensource.com/article/19/5/python-3-default-mac#what-to-do). Up and running now, I've produced a .wav from and image sequence and must say it's surprisingly listenable! :) Haven't tried the round-trip yet...Thanks for this project, anyway. And thanks for your help!
